helm/nginx-ingress-controller-app/templates/controller-poddisruptionbudget.yaml

```diff
@@ -0,0 +1,15 @@
+{{- if or (and .Values.controller.autoscaling.enabled (gt (.Values.controller.autoscaling.minReplicas | int) 1)) (gt (.Values.ingressController.replicas | int) 1) }}
+apiVersion: policy/v1beta1
+kind: PodDisruptionBudget
```

Comment:
With HPA enabled there's additional source of voluntary disruptions. This PDB helps us additionally protect nginx deployment, control how many instances can be unavailable at the same time.